sepal_ui/sepalwidgets/app.py

```diff
@@ -443,6 +437,7 @@ def __init__(
         code: str = "",
         wiki: str = "",
         issue: str = "",
+        repo_folder: str = Path.cwd(),
```

Comment:
your hint expects a str and you serve a Pathlib.Path object. 

looking at the error in the code it seems it's also the source of the bug. Instead I would create somwhere (utils ?) a new Type: 
```python 
Pathlike = Union[str, pathlib.Path] 
```

and reuse it everytwhere we set file/dir names (to stay on the safe side). 

For the default value set it to `""` and fill it with Path.cwd() in the init method. At parsing stage, there is no cwd yet.

Reply:
Done, but I didn't use the `Pathlike` everywhere... perhaps is better if we handle that in a different PR? we could create a `types.py` file (if that's something common)